android/src/main/java/com/imagepicker/VideoMetadata.java

```diff
@@ -0,0 +1,28 @@
+package com.imagepicker;
+
+import static java.lang.Integer.parseInt;
+
+import android.content.Context;
+import android.media.MediaMetadataRetriever;
+import android.net.Uri;
+
+public class VideoMetadata {
+  int duration;
+  int bitrate;
+
+  public VideoMetadata(Uri uri, Context context) {
+    MediaMetadataRetriever m = new MediaMetadataRetriever();
+    m.setDataSource(context, uri);
```

Comment:
Do we want to favour descriptive names over single letter naming (it increases code readability) i.e. 

```suggestion
    MediaMetadataRetriever metadataRetriever = new MediaMetadataRetriever();
    metadataRetriever.setDataSource(context, uri);
```

Reply:
done! commited  🙌